### PR TITLE
[EDGE] Add MQTT data transmission test @open sesame 04/20 12:55

### PR DESCRIPTION
**Pull request by gichan-jang**
Add run test for MQTT data transmission of edgesrc/sink.

Signed-off-by: gichan <gichan2.jang@samsung.com>

**Self evaluation:**
1. Build test: [ ]Passed [ ]Failed [*]Skipped
2. Run test: [ ]Passed [ ]Failed [*]Skipped

### CODE REVIEW

File: gst/edge/edge_common.c [hunk 1]
```diff
@@ -31,6 +31,8 @@ gst_edge_get_connect_type (void)
           "Connect with MQTT brokers and directly sending stream frames via TCP connections."},
       {NNS_EDGE_CONNECT_TYPE_AITT, "AITT",
           "Sending stream frames via AITT connections."},
+      {NNS_EDGE_CONNECT_TYPE_AITT, "MQTT",
```
Comment (songgot) [inline gst/edge/edge_common.c:34]: How about replacing 'NNS_EDGE_CONNECT_TYPE_AITT' with 'NNS_EDGE_CONNECT_TYPE_MQTT' ?

Comment (gichan-jang) [reply gst/edge/edge_common.c:34]: nnstreamer-edge supports both AITT and MQTT pub/sub.

**Review by again4you — APPROVED** [review]
LGTM!

File: gst/edge/edge_common.c (hunk 1 above)
Comment (jaeyun-jung) [inline gst/edge/edge_common.c:34]: plz check enum val : NNS_EDGE_CONNECT_TYPE_AITT > NNS_EDGE_CONNECT_TYPE_MQTT

Comment (gichan-jang) [reply gst/edge/edge_common.c]: Oh.. My bad.. Updated.

**Review by jaeyun-jung — APPROVED** [review]

**Pull request merged**